src/cli/cli.ts

```diff
@@ -72,6 +83,10 @@ export const getHelpText = {
     --output-stdout
        Does not generate an output file and prints the generated code to stdout
        instead. Has precedence over -o/--output.
+
+    --output-type
+       Sets the output language, defaults to Typescript.
```

Comment:
this does not have precedence over automated inference as .js + output-type = TS would get changed to .js + output-type = ESM. Could alternatively declare that an invalid parameter set and error out i guess.

id rather not encourage anyone to use commonjs + typescript lol.

feel free to change the text on this PR if you want. Otherwise ill just copy paste what you write unless i think its inaccurate.